model/src/main/scala/za/co/absa/atum/model/dto/AdditionalDataSubmitDTO.scala

```diff
@@ -0,0 +1,6 @@
+package za.co.absa.atum.model.dto
+
+case class AdditionalDataSubmitDTO (
+  atumPartitioning: PartitioningDTO,
```

Comment:
```suggestion
  partitioning: PartitioningDTO,
```

just to keep it consistent with other DTO objects